recipe/meta.yaml

```diff
@@ -15,83 +17,188 @@ source:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
+    - {{ stdlib('c') }}
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy                               # [build_platform != target_platform]
+    - postgresql                          # [build_platform != target_platform]
+    - python                              # [build_platform != target_platform]
     - cmake
-    - jom  # [win]
+    - jom   # [win]
     - make  # [unix]
     - pkg-config
   host:
-    - libboost-devel
-    - libboost-python-devel
     - cairo
     - eigen
     - freetype
-    - python
+    - libboost-devel
+    - libboost-python-devel
     - numpy
-    - pillow
     - pandas <2.2
+    - pillow
     - pip
+    - postgresql
+    - python
     - setuptools
     - setuptools_scm >=8
-    # For rdkit-stubs
-    - pybind11-stubgen
-  run:
-    - cairo
-    - python
-    - pillow
-    - pandas
-    - {{ pin_compatible('numpy') }}
-    - pycairo
-    - matplotlib-base
-    - sqlalchemy
-    - reportlab
-
-test:
-  commands:
-    - python -c "import rdkit; assert rdkit.__version__ == '{{ version }}'"
-  imports:
-    - rdkit
-    - rdkit.Avalon
-    - rdkit.Chem
-    - rdkit.Chem.AllChem
-    - rdkit.Chem.rdFreeSASA
-    - rdkit.DataManip
-    - rdkit.Dbase
-    - rdkit.DistanceGeometry
-    - rdkit.ForceField
-    - rdkit.Geometry
-    - rdkit.ML
-    - rdkit.Numerics
-    - rdkit.SimDivFilters
-    - rdkit.VLib
-    - rdkit.VLib.NodeLib
-    - rdkit.Chem.PandasTools
 
 outputs:
-  - name: rdkit
-  - name: rdkit-dev
-    script: install_rdkit_dev.bat  # [win]
+  - name: librdkit
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+        - numpy                               # [build_platform != target_platform]
+        - python                              # [build_platform != target_platform]
+        - cmake
+      host:
+        - cairo
+        - eigen
+        - freetype
+        - libboost-devel
+        - libboost-python-devel
+        - matplotlib-base
+        - numpy
+        - pandas <2.2
+        - pillow
+        - pycairo
+        - python
+        - reportlab
+        - sqlalchemy
+      run:
+        - cairo
+        - eigen
+        - freetype
+        # NOTE(skearnes): The `host` dependencies above are only listed so that this pin is correctly resolved. Without
+        # them (or some subset that I haven't narrowed to yet), libboost may be resolved to a higher version than was
+        # used for the build in build.sh.
+        - {{ pin_compatible('libboost') }}
+    about:
```

Comment:
Can we add some tests for this output? Checking the presence of certain libraries is enough.

Reply:
Added some simple checks for an exported library.